### ISSUE TITLE
Animating stroke-dasharray

### ISSUE THREAD

**Issue by domonique-ncino** [svgdotjs/svg.js#698]
There's not a smooth transition when trying to animate a stroke-dasharray attribute.
```css
@keyframes dash{
  0%{
    stroke-dasharray: 1,200;
    stroke-dashoffset: 0;
  }

  100%{
    stroke-dasharray: 214,270;
    stroke-dashoffset: 25;
  }
}
```

Works in CSS
```js
circle.attr({
  cx: frame.x / 2,
  cy: frame.y / 2,
  'fill-opacity': 0,
  stroke: '#fff',
  'stroke-width': 2,
  'stroke-dasharray': [10, 10],
  'stroke-dashoffset': 0,
  'stroke-linecap': 'round'
});

circle.animate(2000, '<>', 0).attr({
  'stroke-dasharray': [1, 1],
  'stroke-dashoffset': 0
}).rotate(360);
```
Not as smooth

**Comment by Fuzzyma** [comment]
Css animation are backed by the graphic card and don't alter the Dom tree. Ofc js is slower and might be not that smooth on slow pcs

**Comment by domonique-ncino** [comment]
So I should keep everything in the css?

**Comment by Fuzzyma** [comment]
This is completely up to you. If you like css animations more use them. There is not much I can do about that :)

**Comment by domonique-ncino** [comment]
I would love to be able to change the keyframes to dynamic fields using javascript. But if I can't, I will stick with CSS

**Comment by domonique-ncino** [comment]
When I run the animate(), the stroke-dasharray doesn't change or you don't see it change in the animation. That's the issue.

**Comment by Fuzzyma** [comment]
There is no animation at all? Well that's unexpected. Do you kind creating a jsfiddle for this css which shows the problem?

**Comment by domonique-ncino** [comment]
The good version: https://jsfiddle.net/79m1cwx9/

Not so good version: https://jsfiddle.net/hmkv19fr/1/

**Comment by Fuzzyma** [comment]
Add this to line number [2041](https://github.com/svgdotjs/svg.js/blob/master/dist/svg.js#L2041)

```js
    // prepare value list for morphing
    if(SVG.regex.delimiter.test(from)) return new SVG.Array(from).morph(to)
```

It should now work. Please report back

**Comment by Fuzzyma** [comment]
@domonique-ncino any news?

**Comment by domonique-ncino** [comment]
Works great! Thank you

**Comment by Fuzzyma** [comment]
fixed with a19b4f1dfe06d70775a03ec5428b0423fbb46819

**Comment by RmiTtro** [comment]
tests added with c9b762e
